schools/management/commands/loadschools.py

```diff
@@ -16,8 +16,8 @@ def query_api(api_endpoint_id):
     doc = requests.get(url, params=api_get_params)
     return doc.json()['features']
 
-def query_api2(api_endpoint_id):
-    api_base = 'http://gisweb2.ci.durham.nc.us/arcgis/rest/services/DurhamMaps/DPS_ElementaryStudentAssignment/MapServer'
+def query_api2(api_endpoint_id, api_section):
```

Comment:
I think we could make this line:

    def query_api2(api_endpoint_id, api_section='DPS_ElementaryStudentAssignment'):

so that we can remove lines: 97, 108, and 120.